tests/stdin_test.c

```diff
@@ -149,7 +149,11 @@ stdin_test	(int typemajor, int count)
 	else
 		memset (&sfinfo, 0, sizeof (sfinfo)) ;
 
+#ifdef STDIN_FILENO
 	if ((file = sf_open_fd (STDIN_FILENO, SFM_READ, &sfinfo, SF_TRUE)) == NULL)
+#else
+	if ((file = sf_open_fd (fileno (stdin), SFM_READ, &sfinfo, SF_TRUE)) == NULL)
```

Comment:
That should work on Linux and OS X. Probably better to swap to that for all platforms.